librecad/src/lib/engine/rs_ellipse.cpp

```diff
@@ -799,6 +799,58 @@ bool RS_Ellipse::createFromQuadratic(const QVector<double>& dn){
 }
 
 /**
+ * The general equation of a conic is Ax^2 + Bxy + Cy^2 + Dx + Ey + F = 0
+ * In matrix form it is: {x, y}.m_mQuad.{{x},{y}} + m_vLinear.{{x},{y}}+m_dConst=0
+ * If the det(m_mQuad) < 0,  then the conic is an ellipse.
+ * This implies that B = 0 and equation becomes Ax^2 + Cy^2 + Dx + Ey + F = 0, A != 0, C != 0, A != C, 
```

Comment:
No, the requirement of the equation being an ellipse does NOT lead to B=0, for example,  an ellipse at 135 degree majorP could be: 4(x+y)^2 + （x-y)^2 = 5 x^2 + 2xy + 5 y^2 =1, with its determinant = 24 > 0